howto/android/custom-vhal.md

```diff
@@ -136,6 +136,100 @@ addons:
   - custom-vhal
 ```
 
+# Integrate Anbox VHAL interface
```

Comment:
```suggestion
## Integrate Anbox VHAL interface
```

Reply:
+1. With the integration of the Anbox VHAL interface, the page 'Customize the Anbox VHAL' becomes complete. Originally, the page was more focused on side-loading a VHAL implementation and making it work with the Anbox runtime via an addon.